index.js

```diff
@@ -14,11 +14,12 @@ const moons = [
 
   You should not need to make any other changes to the code.
 */
-function organizeMoonsByPlanet(moons) {
+const organizeMoonsByPlanet = (moons) =>{
   const result = {};
   for (let moon of moons) {
-    const name = moon.name;
-    const planet = moon.planet;
+
+    const {name, planet} = moon 
```

Comment:
I would encourage you to use semi-colons at the ends of your lines in code that's submitted professionally (which this is). It will prevent edge cases that _completely_ change how your code is run, like the following:

```js
const newAges = people.map((person) => person.age)

[...newAges, ...ages].forEach((age) => console.log(age))
```

Without the semi-colon, that is read as one line, which means that the mapped array from the first line is then accessed at the _index_ [...newAges, ...ages], which will be a syntax error.

This also happens if you start a line with a parenthesis without ending the previous line with a semi-colon. Then the lines are combined and the last thing on the previous line has parentheses after it, making it an _accidental function call_.

To avoid these bugs and to ensure your code looks professional, add the semi-colons before a `git add`, or get in the habit of writing them to start with.